tests/conftest.py

```diff
@@ -67,7 +67,7 @@ def containers(request, confdir):
     if request.config.option.usedocker:
         docker = request.getfixturevalue('dockerctl')
         with docker.run(
-            'safarov/freeswitch',
+            'safarov/freeswitch:1.10.3',
```

Comment:
Could we move this to a setting, please?

Reply:
ok, sure.